2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,77 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+// First Recipe
+const recipe1 = {
+  title: "Mole",
+  servings: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"]
+};
```

Comment:
So this looks good, no issues with your objects. But you could try to do this as a nested object instead?

Reply:
Will do!